apps/catalogue/src/views/VariableView.vue

```diff
@@ -149,6 +149,7 @@
             <tr>
               <th>fromTable</th>
               <th>fromVariables</th>
+              <th>fromVariablesInOtherTables</th>
```

Comment:
The whole things seems confusing to me. By trying to force a ono-to-one mapping of model and interface the storage details bleed into the user interface. Suggest writing down who is using the screen to accomplish what tasks. Then setup the interface to support this task. Instead to visualising the storage model.

Reply:
in this case we could do this indeed. You may have noticed that in other parts of the UI I indeed merged them. Good suggestion to do that here too.

Reply:
NB, I disagree with the overall principele that there shouldn't be one-to-one mapping of model and interface. This is scientific software and there the data submitters are just (or even more) important then the users. Hence a much higher transparancy is required in terms on what actually is being stored (as compared to consumer software were indeed you should often hide the underlying model)

Reply:
done